### ISSUE TITLE
Generation process fails if asset contains any sizes without icons

### ISSUE THREAD

**Issue by iKenndac** [alexaubry/alternate-icons#1]
By default, Xcode will create icon assets with many sizes, not all of which will be filled. For instance, CarPlay and "iTunes Marketing" are created in Xcode 9, which there's no need to fill with images for app icons in most cases.

If you don't fill sizes with images, the `filename` key will be missing entirely:

```
    {
      "size" : "40x40",
      "idiom" : "iphone",
      "scale" : "2x"
    }
```

When encountering such items, the app fails with the following error: `💥  The operation couldn’t be completed. (Unbox.UnboxError error 0.)`

The only way to get around this is to manually strip these parts out using a text editor. Unfortunately, since these are valid as far as Xcode is concerned, they empty sections will be re-added if you open the file again with Xcode.

**Comment by alexaubry** [comment]
Thanks for pointing out 😄 This bug has been fixed in [v1.1](https://github.com/alexaubry/alternate-icons/releases/tag/1.1.0)

**Comment by iKenndac** [comment]
Awesome! This is such a useful tool — thanks for making it! :)
